### ISSUE TITLE
Python user on Google Groups

### ISSUE THREAD

**Issue by wibeasley** [sburns/PyCap#56]
@sburns, Consider if there's something you want to add to this thread
https://groups.google.com/forum/#!topic/project-redcap/p-9v51TbK6o

**Comment by sburns** [comment]
How do you join that group?

**Comment by wibeasley** [comment]
Oh man, you were really working with a hand tied behind your back if you weren't a member of that group while you were developing PyCap.  It's the main channel for people on different campuses helping out each other.  The group admins at Vandy prefer that typical REDCap users don't have access to it --only site admins.  I think they prefer that you petition your site's admin for access to the group, instead of doing it directly.

If you're not planning to do much with PyCap anymore, I wouldn't worry about it.  If you are, consider requesting access, on the grounds that you're maintaining the only Python library for the REDCap API.

For that particular thread that I linked to, someone was having trouble with the API, and they happened to be using Python.  I posted that they consider PyCap.  I think it was a proof of concept that they eventually solved it with PHP.

>  This group is for technical discussions related to installing and maintaining REDCap and administering REDCap projects at consortium partner sites. http://project-redcap.org/
